tests/grpcio-wheel/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.poetry]
```

Comment:
Should we share the `pyproject.toml`/`.lock` files across wheel/no-wheel, to reduce chances that they diverge over time?

Reply:
That sounds good, but I'll just remove the `no-wheel` test for now.